flink-connector/flink/build.gradle.kts

```diff
@@ -98,6 +98,8 @@ dependencies {
   testImplementation(libs.testcontainers.mysql)
   testImplementation(libs.metrics.core)
 
+  testImplementation("org.apache.iceberg:iceberg-core:$icebergVersion")
```

Comment:
any reason to add these two packages?

Reply:
An error will occur  https://github.com/apache/gravitino/actions/runs/13792221625/job/38574823127